arm-sql/2015-05-01-preview/swagger/automaticTuning.json

```diff
@@ -0,0 +1,491 @@
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "ServerAutomaticTuning"
+        ],
+        "description": "Update default automatic tuning options on server.",
+        "operationId": "ServerAutomaticTuning_Update",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ServerNameParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "description": "The requested automatic tuning resource state.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/AutomaticTuningUpdate"
+            }
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully updated the default server-level automatic tuning options.",
+            "schema": {
+              "$ref": "#/definitions/AutomaticTuningUpdate"
+            }
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 400 InvalidIdentifier - The identifier contains NULL or an invalid unicode character.\n\n * 400 InvalidAutomaticTuningUpsertRequest - \n\n * 400 InvalidAdvisorAutoExecuteStatus - Specified auto-execute status for the advisor is not allowed.\n\n * 404 SubscriptionDoesNotHaveServer - The requested server was not found\n\n * 404 DatabaseDoesNotExist - The requested database was not found\n\n * 404 SubscriptionDoesNotHaveElasticPool - The requested elastic pool was not found\n\n * 404 OperationIdNotFound - The operation with Id does not exist.\n\n * 405 NotSupported - This functionality is not supported.\n\n * 409 Conflict - Request could not be processed because of conflict in the request.\n\n * 409 OperationCancelled - The operation has been cancelled by user.\n\n * 409 OperationInterrupted - The operation on the resource could not be completed because it was interrupted by another operation on the same resource.\n\n * 500 OperationTimedOut - The operation timed out and automatically rolled back. Please retry the operation.\n\n * 503 ServiceTemporarilyUnavailable - Feature temporarily unavailable."
+          }
+        }
+      }
+    }
```

Comment:
json is not valid - missing "comma" https://travis-ci.org/Azure/azure-rest-api-specs/jobs/229880407#L2188